test/core/image_processing/median_filter.cpp

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright 2019 Miral Shah <miralshah2211@gmail.com>
+//
+// Use, modification and distribution are subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+//
+
+#define BOOST_TEST_MODULE test_image_processing_median_filter
+#include "unit_test.hpp"
+
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/algorithm.hpp>
+#include <boost/gil/gray.hpp>
```

Comment:
Nitpicking for future:  these should be kept in order https://github.com/boostorg/gil/wiki/Include-Directives-Order